packages/ui/readme.md

```diff
@@ -17,3 +17,111 @@ Your on-ramp to web3 multichain. Web3Modal is a versatile library that makes it
 ## Development
 
 Please follow [developer docs](./.github/docs/development.md) to set up web3modal locally.
+
```

Comment:
We need to put these docs under `.github/docs` somewhere as package readme's are user facing / displayed on npm.

Reply:
Good idea, I will move it